<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="ProdtableWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #C0C0C0;
alternate-background-color: #606060;
selection-background-color: #282828;
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Наименование</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="OrdtableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>331</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #C0C0C0;
alternate-background-color: #606060;
selection-background-color: #282828;
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Реквизиты</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="NumberNamelabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер заказа№:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumberTextlabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PriceNamelabe">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Итоговая цена</string>
    </property>
   </widget>
   <widget class="QLabel" name="PriceTextlabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SavepushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить в</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadpushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить из</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SwitchcomboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Заказы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Товары</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="KeybordpushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить С клавиатуры</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="OrderNumspinBox">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>290</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="Ordlabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер заказа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadpushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Список товаров</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangepushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProductNumberlabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер товара</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ProdNumspinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="DeletepushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <zorder>ProdtableWidget</zorder>
   <zorder>OrdtableWidget</zorder>
   <zorder>NumberNamelabel</zorder>
   <zorder>NumberTextlabel</zorder>
   <zorder>PriceNamelabe</zorder>
   <zorder>PriceTextlabel</zorder>
   <zorder>SavepushButton</zorder>
   <zorder>LoadpushButton</zorder>
   <zorder>KeybordpushButton</zorder>
   <zorder>SwitchcomboBox</zorder>
   <zorder>OrderNumspinBox</zorder>
   <zorder>Ordlabel</zorder>
   <zorder>LoadpushButton_2</zorder>
   <zorder>ChangepushButton</zorder>
   <zorder>ProductNumberlabel</zorder>
   <zorder>ProdNumspinBox</zorder>
   <zorder>DeletepushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
